moped-editor/src/queries/signals.js

```diff
@@ -4,20 +4,9 @@ export const SIGNAL_PROJECTS_QUERY = gql`
   query SignalProjectsQuery {
     moped_project(
       where: {
-        is_deleted: { _eq: false }
-        _or: [
-          {
-            moped_proj_components: {
-              moped_components: { component_name: { _ilike: "signal" } }
-              is_deleted: { _eq: false }
-            }
-          }
-          {
-            moped_proj_components: {
-              moped_components: { component_name: { _ilike: "signal" } }
-              is_deleted: { _eq: false }
-            }
-          }
+        _and: [
+          { is_deleted: { _eq: false } }
+          { moped_proj_components: { feature_signals: {} } }
```

Comment:
this is a slight change in behavior—the project must have a signal component _feature_, whereas before the project merely needed a signal component. it should have an affect from the user perspective and seemed more explicit?

Reply:
This makes sense and looks more straightforward to me! I did think of one thing while poking around. Is a signal project still considered a signal project if a user deletes the signal component that was originally added to the project? 

I keep getting hung up on how `is_deleted` in the `features` table factors into all of this. I hope that makes sense.

Reply:
oof—yes, you're totally right that the this query needs to consider `is_deleted`. i think hasura should be able to handle this but i'm going to have to wrap my brain around it first. thanks for flagging this!